service/src/main/java/zone/cogni/asquare/service/elasticsearch/info/ElasticsearchMetadata.java

```diff
@@ -122,4 +122,26 @@ public int hashCode() {
     }
   }
 
+  public static class Configuration {
+
+    private int connectTimeout;
+    private int readTimeout;
+
+    public int getConnectTimeout() {
+      return connectTimeout;
+    }
+
+    public void setConnectTimeout(int connectTimeout) {
```

Comment:
maybe set a bit a comment here (main info i want here is the unit, seconds/milliseconds?)